back/app/models/custom_field.rb

```diff
@@ -292,8 +292,12 @@ def ordered_options
   def linear_scale_print_description(locale)
     return nil unless linear_scale?
 
-    min_label = "1#{minimum_label_multiloc[locale].present? ? " (#{minimum_label_multiloc[locale]})" : ''}"
-    max_label = maximum.to_s + (maximum_label_multiloc[locale].present? ? " (#{maximum_label_multiloc[locale]})" : '')
+    min_text = linear_scale_label_1_multiloc[locale]
+    min_label = "1#{min_text.present? ? " (#{min_text})" : ''}"
+
+    max_text = nth_linear_scale_multiloc(maximum)[locale]
+    max_label = maximum.to_s + (max_text.present? ? " (#{max_text})" : '')
+
```

Comment:
Aware this was there already, but it's better to use `MultilocService.t()` in stead of `[locale]`. It has fallback and error logic built in.